config/locales/fr.yml

```diff
@@ -1,6 +1,8 @@
 fr:
   language_name: "Français"
   activerecord:
+    models:
+      spree/product: Produit
```

Comment:
This is helpful for testing but can you remove it again? Otherwise we'll have merge conflicts with our translation process.

Reply:
sure i will remove it